docusaurus/docs-internals/editor-controls/multivalue-controls/typography.md

```diff
@@ -0,0 +1,316 @@
+---
+toc_max_heading_level: 4
+---
+# Typography
+
+The `typography` control provides a comprehensive set of features for manipulating text on a web page. It supports loading fonts from Google Fonts or custom sources, altering CSS properties related to text appearance (such as strikethrough, bold, italic, underline, uppercase, and lowercase), and selecting or adding fonts from global styles. Additionally, it allows for adjusting text size, weight, line height, and letter spacing.
+
+Example of the control:
+
+![Typography](/img/multivalue-controls/typography.png)
+
+### Parameters
```

Comment:
`icons` is missing in params
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/da8f6b97-fe46-4a4b-824c-72c1462b78de)